components/bitcoin-topics/layouts/TopicLayout.tsx

```diff
@@ -9,15 +10,24 @@ import { PrevNextLinks } from "../topic/PrevNextLinks"
 import { Prose } from "../topic/Prose"
 import { TopicHeader } from "../topic/TopicHeader"
 import { usePathname } from "next/navigation"
+import {
+    motion,
+    AnimatePresence,
+    useMotionValue,
+    useTransform,
+    useSpring,
+    PanInfo
+} from "framer-motion"
+import { ChevronRight } from "lucide-react"
 
-const editUrl = (path: string) =>
+const editUrl = (path: string): string =>
```

Comment:
`editUrl` is never used

Reply:
nice catch, I added a new button to suggest edits for a topic